gql/transport/websockets.py

```diff
@@ -175,6 +175,23 @@ async def _receive(self) -> str:
 
         return answer
 
+    async def _wait_ack(self) -> None:
+        """Wait for the connection_ack message. Keep alive messages are ignored
+        """
+
+        while True:
+            init_answer = await self._receive()
+
+            answer_type, answer_id, execution_result = self._parse_answer(init_answer)
+
+            if answer_type == "connection_ack":
+                return
+
+            if answer_type != "ka":
+                raise TransportProtocolError(
+                    "Websocket server did not return a connection ack"
+                )
```

Comment:
Shouldn't be `"Webscocket server did not return a keep-alive"` ?

Reply:
No, we are waiting here for a connection_ack (just above)
If we don't receive a connection_ack, we check if it was instead a keep-alive.
If it is not a keep-alive here, it means that it is not a keep-alive and not a connection_ack so we raise the exception same as before.
If it is a keep-alive, then we continue in the loop and will listen to the next message received hoping to receive a connection_ack next.

Reply:
Fair enough, got it wrong while reviewing it.